src/scheduler/pbsfs.c

```diff
@@ -141,23 +141,58 @@ main(int argc, char *argv[])
 		exit(1);
 	}
 	if ((flags & (FS_PRINT | FS_PRINT_TREE)) && (argc - optind) != 0) {
-		fprintf(stderr, "Usage: pbsfs -[ptdgcs]\n");
+		fprintf(stderr, "Usage: pbsfs -[ptdgcs] [-I sched_name]\n");
 		exit(1);
 	}
 	else if ((flags & FS_GET)  && (argc - optind) != 1) {
-		fprintf(stderr, "Usage: pbsfs -g <fairshare_entity>\n");
+		fprintf(stderr, "Usage: pbsfs -i [sched_name] -g <fairshare_entity>\n");
 		exit(1);
 	}
 	else if ((flags & FS_SET) && (argc - optind) != 2) {
-		fprintf(stderr, "Usage: pbsfs -s <fairshare_entity> <usage>\n");
+		fprintf(stderr, "Usage: pbsfs -I [sched_name] -s <fairshare_entity> <usage>\n");
```

Comment:
@bhroam  shouldn't -I be inside the square brackets? right now it looks like -I is necessary but it may or may not need scheduler name